packages/building-rollup/README.md

```diff
@@ -263,7 +263,42 @@ export default merge(basicConfig, {
 });
 ```
 
-#### Resolve commonjs modules
+#### Deduplicating Node Modules
+
+Rollup uses the browser's standard means of resolving modules by URL. That means that by default, bare module specifiers are not supported. As mentioned [above](#config-features), Both the modern and legacy open-wc configs apply `rollup-plugin-node-resolve` which allows you to use bare specifiers in your source files.
+
+Node module resolution is fundamentally different to browser behaviour. Node resolution allows - and in fact encourages - multiple copies of single modules to exist in your app's module graph. This can cause issues for libraries which expect to be loaded only once by the browser. `lit-html` is one such library, which uses [`instanceof`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof) extensively throughout the source. If it's loaded twice, [bugs can occur](https://github.com/Polymer/lit-html/issues/320#issuecomment-390768466).
+
+To avoid this problem, we've included `lit-html` in the [list of packages for `rollup-plugin-node-resolve` to deduplicate](https://github.com/rollup/rollup-plugin-node-resolve/pull/201). You can customize this list yourself by adding your module names to the `dedupeModules` option of the config generator.
+
+```js
```

Comment:
The convention for this right now is to add a boolean to the object passed to the config creator to turn off a plugin. This is a pretty nice way as well, but we should be consistent here.

Reply:
So i'll add a check for `dedupe === false`. However, I think this paragraph should stay in the docs, in case a user has some weird reason for wanting to override the defaults but still dedupe. Is that unreasonable?

Reply:
They would pass false, and then add their own version of the plugin I think